pkg/core/hooks/conn/conn.go

```diff
@@ -34,17 +35,29 @@ type ID struct {
 //	    s64 validate_rd_bytes
 //	    s64 validate_wr_bytes
 //	};
-type SocketDataEvent struct {
+type SocketDataEventBig struct {
+	EntryTimestampNano   uint64
+	TimestampNano        uint64
+	ConnID               ID
+	Direction            TrafficDirectionEnum
+	MsgSize              uint32
+	Pos                  uint64
+	ValidateReadBytes    int64
```

Comment:
Can this be done in a way - `type SocketDataEventstruct {
	EntryTimestampNano   uint64
	TimestampNano        uint64
	ConnID               ID
	Direction            TrafficDirectionEnum
	MsgSize              uint32
	Pos                  uint64
	ValidateReadBytes    int64
	ValidateWrittenBytes int64
	ClientID             uint64
}

type SocketDataEventBig struct {
	SocketDataEvent
	Msg [EventBodyMaxSizeBig]byte
}

type SocketDataEventSmall struct {
	SocketDataEvent
	Msg [EventBodyMaxSize]byte
}
`

this one seems to be more readable